### PR TITLE
Algorithm for Catalan Number (DP)

### PR DESCRIPTION
**Pull request by archie94**

### CODE REVIEW

File: algorithm/dp/catalan_number/desc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "Catalan Number": " Pascal's triangle is a triangular array of the binomial coefficients.",
```
Comment (faviouz) [inline algorithm/dp/catalan_number/desc.json]: Description is wrong here

**Comment by duaraghav8** [comment]
Thanks for pointing out @faviouz 
@archie94 Please correct the description & make a PR.

**Comment by archie94** [comment]
Sorry about that ! My bad . Thanks @faviouz

**Comment by archie94** [comment]
Done in #116 ! Thanks again @faviouz

**Pull request merged**